cockroachcloud/cloud-org-sso.md

```diff
@@ -0,0 +1,126 @@
+      - After you enable Cloud Organization SSO, all other members of your {{ site.data.products.db }} organization who were using [Basic SSO](cloud-org-sso.html#basic-sso) rather than an email and password must sign in again to regain access to your organization. After signing in, members retain the same access they had before the migration.
+
+During enablement of the feature, a list of affected members is shown, and those members are also notified individually.
+
+### Enforce a requirement to use SSO
+
+Using SSO with a centralized identity management and user provisioning is a recommended security practice. It helps to mitigate against the risk of unauthorized access to your organization's data and infrastructure by users who should no longer have access, such as when a {{ site.data.products.db }} account using password authentication is not deprovisioned when an employee leaves the organization.
+
+To help mitigate against this risk, SSO is preferred over password authentication, whether your organization uses Basic SSO or Cloud Organization SSO. If a member opts to log in using SSO, their default authentication method is automatically and their previous password is not retained. After a member switches to using SSO, they cannot switch back to using a password, but they can switch to a different SSO authentication method as long as the associated email address is identical. For a member to switch back to using a password, they must be removed and re-invited to your {{ site.data.products.db }} organization.
+
+Refer to [Require SSO](configure-cloud-org-sso.html#require-sso).
+
+## Frequently Asked Questions (FAQ)
+
+#### Will it work if I try to sign in with SSO without explicitly setting it up for my account, if I already use an email from an SSO Provider such as Gmail?
+
+Yes, as long as the email address associated with your SSO provider is exactly the same as the one associated with your existing {{ site.data.products.db }} account. After successfully signing in, you will be prompted to approve the updated authentication method for your account.
+
+To view your current authentication method, visit **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud).
+
+#### Once I change my active login method to a new SSO provider, can I still sign in using my email/password combination or my GitHub identity?
+
+No. Only one authentication method can be active for each {{ site.data.products.db }} Console user. Visit **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) to view or update your active authentication method.
```

Comment:
```suggestion
No. Only one authentication method can be active for each {{ site.data.products.db }} Console user. Users should click **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) to view or update their active authentication method.
```

Reply:
Also, isn't this only true for Basic SSO or did I misunderstand?

Reply:
Yeah, that's what I just commented above :)

Reply:
Addressed by editing Ryan's edit.